### ISSUE TITLE
Proposal discussion > reply to post doesn't show up

### ISSUE THREAD

**Issue by ivanturlakov** [Joystream/pioneer#3967]
Tested on https://pioneer-2-git-mainnet-joystream.vercel.app/#/proposals/preview/47
wss://34.230.5.182.nip.io/ws-rpc

While testing [this issue](https://github.com/Joystream/pioneer/issues/3236)
I entered text at the modal but it doesn’t show up

<img width="913" alt="Снимок экрана 2022-12-10 в 13 56 12" src="https://user-images.githubusercontent.com/46903215/206853978-86a69574-6370-4aa2-b999-82a233039ac0.png">

**Comment by traumschule** [comment]
Confirmed. Likely a modal bug. [This post](https://pioneer-2-git-fork-traumschule-fix-reply-3967-joystream.vercel.app/#/proposals/preview/47?post=16) should show text and QN has `text: ""`:

![post-alice](https://user-images.githubusercontent.com/31551045/206930967-1f5f1b8f-c21f-4d02-b428-50927d45fe5f.png)

https://34.230.5.182.nip.io/query-node/server/graphql
`query { proposalDiscussionPosts (where: { discussionThread: { id_eq: 47 } }, orderBy: createdAt_DESC) { id discussionThreadId authorId text repliesTo { id authorId text } } }`
```
 "data": {
    "proposalDiscussionPosts": [
      {
        "id": "16",
        "discussionThreadId": "47",
        "authorId": "13",
        "text": "",
        "repliesTo": {
          "id": "4",
          "authorId": "24",
          "text": "0 comment"
        }
      },
      {
        "id": "15",
        "discussionThreadId": "47",
        "authorId": "35",
        "text": "",
        "repliesTo": {
          "id": "10",
          "authorId": "13",
          "text": ""
        }
      },
      {
        "id": "14",
        "discussionThreadId": "47",
        "authorId": "29",
        "text": "",
        "repliesTo": {
          "id": "5",
          "authorId": "13",
          "text": "1 comment"
        }
      },
```


![alice-post-event](https://user-images.githubusercontent.com/31551045/206931159-de340713-898d-4940-8710-023d4431968a.png)
[event details](https://polkadot.js.org/apps/?rpc=wss://34.230.5.182.nip.io/ws-rpc#/explorer/query/0x52bba956de131ccac42c9465459513110797bf97d8fd43a6e9c600be77e88942)

**Comment by traumschule** [comment]
@mkbeefcake assigned according to [your message](https://discord.com/channels/811216481340751934/812344874099277824/1057607316922441748) although i realize you got a quite a few already.
> You have postText in pioneer/packages/ui/src/forum/modals/PostActionModal/CreatePostModal/CreatePostModal.tsx. As i understand it this has to be sent through bytesToString before sending to the chain.

**Comment by mkbeefcake** [comment]
https://pioneer-2-git-mainnet-joystream.vercel.app/#/forum/thread/34
wss://34.230.5.182.nip.io/ws-rpc

I have checked it in testnet with latest dev branch, it looks like working well...

**Comment by thesan** [comment]
Fixed in #4023

**Comment by ivanturlakov** [comment]
✅ Tested on https://pioneerapp.xyz/#/proposals/preview/33

<img width="910" alt="Снимок экрана 2023-01-09 в 16 03 16" src="https://user-images.githubusercontent.com/46903215/211326082-19711216-7bed-4169-affd-a32c470977c2.png">
